src/test/java/org/fogbowcloud/manager/core/threads/TestOpenProcessor.java

```diff
@@ -0,0 +1,360 @@
+package org.fogbowcloud.manager.core.threads;
+
+import java.util.Properties;
+
+import org.fogbowcloud.manager.core.constants.ConfigurationConstants;
+import org.fogbowcloud.manager.core.instanceprovider.InstanceProvider;
+import org.fogbowcloud.manager.core.models.orders.ComputeOrder;
+import org.fogbowcloud.manager.core.models.orders.Order;
+import org.fogbowcloud.manager.core.models.orders.OrderRegistry;
+import org.fogbowcloud.manager.core.models.orders.OrderState;
+import org.fogbowcloud.manager.core.models.orders.UserData;
+import org.fogbowcloud.manager.core.models.orders.instances.OrderInstance;
+import org.fogbowcloud.manager.core.models.token.Token;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+public class TestOpenProcessor {
+
+	private OpenProcessor openProcessor;
+
+	private InstanceProvider localInstanceProvider;
+	private InstanceProvider remoteInstanceProvider;
+
+	private OrderRegistry orderRegistry;
+
+	private Properties properties;
+
+	@Before
+	public void setUp() {
```

Comment:
I'm very bad at reading test code. Do you mind adding internal doc lines saying what is going to be tested?

something like that:
//initial conditions
//your call
//what is expected

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/20b9616563e4eef210ba90858eda6b3c3ea1f16a).